apps/api/e2e/tests/price/multi-chain-provider.test.ts

```diff
@@ -110,18 +112,14 @@ describe("Multi-Chain Provider Tests", () => {
       return;
     }
 
-    // The specific_chain column should already be added by the dbManager.initialize()
-    // Let's just verify it exists
-    const pool = dbManager.getPool();
-
-    const result = await pool.query(`
-      SELECT EXISTS (
+    const result = await db.execute(
+      sql`SELECT EXISTS (
         SELECT 1
         FROM information_schema.columns
         WHERE table_name = 'prices'
         AND column_name = 'specific_chain'
-      ) as column_exists;
-    `);
+      ) as column_exists;`,
```

Comment:
Curious why we use `db.execute` here instead of `findFirst`?  Is this just a better performing query?

Reply:
We're querying the postgres information catalog here, so they are tables that we don't define in our Drizzle schema. The idea is to see if a table that we do create is actually created. No way to access them using the typical Drizzle api that exists for the table we define ourselves. 

This test is probably not needed now that we have Drizzle. We migrate the database using Drizzle to ensure all our tables are created. If that process fails, the test setup will fail and we won't even get to the point of running tests. I'll delete this test.

Reply:
Deleted the test.